subset/switches/src/main/java/switchtest/SwitchInterrogator.java

```diff
@@ -48,22 +50,66 @@
     "enable",
     "show interface port1.0.",
     "show platform port port1.0.",
+    "show power-inline interface port1.0.",
     "show run",
-    "show stack",
-    "show power inline power-consumption",
+    "show stack"
   };
 
   String[] commandToggle = {"interface ethernet port1.0.", "shutdown", "no shutdown"};
 
   int interfacePos = 1;
   int platformPos = 2;
+  int powerinlinePos = 3;
```

Comment:
@ewRedstone This should be an enum

Reply:
please check the use case, this is referring to a position of a command which is referenced for parsing throughout.